pkgs/by-name/gy/gysmo/package.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, go }:
+
+stdenv.mkDerivation rec {
+  pname = "gysmo";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "grosheth";
+    repo = "gysmo";
+    rev = "v${version}";
+    sha256 = "ucDcEygh315yD999z6W+kqNBZLj0lGuxZUYjSkOhcJw=";
+  };
+
+  buildInputs = [ go ];
+
+  buildPhase = ''
+    export GOCACHE=$TMPDIR/go-build
+    export HOME=$TMPDIR/home
+    go build -mod=vendor -o gysmo src/main.go
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp gysmo $out/bin/
+
+    cp -r src/ascii $out/
+    cp -r src/config $out/
+  '';
+
+  meta = with lib; {
+    description = "A tool to display system information with a focus on customization and aesthetics";
```

```suggestion
    description = "Display system information with a focus on customization and aesthetics";
```

In nixpkgs, package descriptions are expected not to start with definite (“The”) or indefinite (“A”/“An”) article. See https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes